src/test/java/com/cloudant/tests/AttachmentsTest.java

```diff
@@ -51,7 +51,6 @@ public void setUp() {
     @After
     public void tearDown() {
         account.deleteDB("lightcouch-db-test");
-        account.shutdown();
```

Comment:
A bunch of these test changes won't be necessary if the `shutdown()` method remains in place.